src/CommitmentsV2/SFA.DAS.CommitmentsV2.Shared.UnitTests/Services/CookieStorageServiceTests/WhenCallingCreate.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.IO;
+using AutoFixture.NUnit3;
+using Microsoft.AspNetCore.DataProtection;
+using Microsoft.AspNetCore.Http;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.CommitmentsV2.Shared.Services;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.CommitmentsV2.Shared.UnitTests.Services.CookieStorageServiceTests
+{
+    public class WhenCallingCreate
+    {
+        [Test, MoqAutoData, Ignore("wip")]
+        public void Then_Stuff(
```

Comment:
There's some oddly named things in this test. There also doesn't appear to be a pass/fail condition

Reply:
Ah this is some work left by Scott when he was trying to create tests for the cookie service. Just spoke with Dan and we are going to remove this code for now so i'll delete it.